ern-core/src/android.ts

```diff
@@ -406,3 +405,39 @@ export function androidEmulatorPath(): string {
   }
   return 'emulator'
 }
+
+/**
+ * Returns the default Hermes engine (hermes-engine) package version used by a given React Native version.
+ * Only works for versions of RN >= 0.60.0 as hermes-engine package was introduced in this version.
+ */
+export function getDefaultHermesVersion(
+  reactNativeVersion: string
+): string | never {
+  if (semver.gte(reactNativeVersion, '0.62.0')) {
+    return '~0.4.0'
+  } else if (semver.gte(reactNativeVersion, '0.60.0')) {
+    return '^0.2.1'
+  } else {
+    throw new Error(
+      'This function can only be called for versions of React Native >= 0.60.0'
+    )
+  }
+}
+
+/**
+ * Returns the default JavaScript Core engine version used
+ * by a given React Native version.
+ * Only works for versions of RN >= 0.60.0 as dynamic jsc-android
+ * pakcage was introduced in this version.
+ */
+export function getDefaultJSCVersion(
+  reactNativeVersion: string
+): string | never {
+  if (semver.gte(reactNativeVersion, '0.60.0')) {
+    return '245459'
```

Comment:
Q1: Do you want to return the full version number (i.e. `245459.0.0` instead of just the major version), to keep it consistent with what `getDefaultHermesVersion` returns?

Q2: It appears starting with RN 0.61, the `jsc-android` dependency is declared as `^245459.0.0` (i.e. _including_ the caret). Do you want to account for that here too?

```
$ yarn info react-native@0.61.0 dependencies | grep 'jsc-android'
  'jsc-android': '^245459.0.0',
```

Reply:
Yes that's a good call.
Will update version resolution in another PR dedicated to this.

Reply:
Sounds good. You mean a separate small PR that just adds the new `getDefaultX` methods _before_ this PR right?

Reply:
Nevermind, just updated the code in this PR as it introduced these methods and it wasn't too much work. Got too lazy to extract the code from this PR to another one ;)

Reply:
OK cool, and what I also meant in the first comment, is to _add an additional_ conditional branch, to keep it consistent with RN:

```
$ yarn info react-native@0.60.0 dependencies | grep 'jsc-android'
  'jsc-android': '245459.0.0',
```

```
$ yarn info react-native@0.61.0 dependencies | grep 'jsc-android'
  'jsc-android': '^245459.0.0',
```

```
if (semver.gte(reactNativeVersion, '0.60.0')) {
  return '245459.0.0'
} else if (semver.gte(reactNativeVersion, '0.61.0')) {
  return '^245459.0.0'
}
```

Reply:
Ok didn't realized they added caret for 61. Will add it 👍

Reply:
will revert conditional order statement though otherwise it'll pick first one for 61 ;)

Reply:
done :)

Reply:
Oops, yes of course 🤦 😄 👍